CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+
+### Added
+
+### Changed
+
+- Remove the audio video observers in the `audioVideoDidStop()` function instead of `leave()` function.
```

Comment:
Can you extend this sentence to end with "in MeetingManager"? We need builders to understand the scope of these functions hence asking.

Reply:
Yep, sure